guides/app-developers/receive-tokens.md

```diff
@@ -54,7 +55,7 @@ export PRIVATE_KEY=''# input your own pk
 cast send 0x274a6990dE7AaE06452cbEFa266c0C6a568F0D5B 'drop()' --private-key $PRIVATE_KEY -r $L1_RPC_URL
 ```
 
-After that you can cross the claimed `Custom Gas Token` to L2 via [entrance](https://quarkchain-b1ac26e1bc5a3c1f.testnets.rollbridge.app/) or follow the instructions [here](https://github.com/ethereum-optimism/specs/discussions/140#discussioncomment-9426636).
+After that you can cross the claimed `Test QKC` to L2 via [entrance](https://quarkchain-b1ac26e1bc5a3c1f.testnets.rollbridge.app/) or follow the instructions [here](https://github.com/ethereum-optimism/specs/discussions/140#discussioncomment-9426636).
```

Comment:
The bridge UI for the devnet is no longer operational. We might need to provide instructions guiding developers on how to use the cast tool to perform bridging instead?

Reply:
Done in c11e770ec5d55a44aa3de87ae836c9b0e1eacf15.